day_6/exercises/dog.js

```diff
@@ -23,12 +23,16 @@ class Dog {
   eat() {
     this.hungry = false;
   }
+
+  play() {
+    this.hungry = true;
```

Comment:
Nice work on this method! It makes sense that we'd make a dog hungry, when they play.